client/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.4.6] - 2021-05-05
+
+- Added the export of `generateJWE`. This was missing from `0.4.4`
```

Comment:
```suggestion
- Added the export of `generateJWE`. This was missing in the `0.4.4` version.
```
You were too quick to make the change ;)

Reply:
😄